jme3-core/src/main/java/com/jme3/util/TestableExecutor.java

```diff
@@ -91,11 +91,17 @@ public static void launch(String javaPackage, Object userData, String[] signatur
             IllegalAccessException, NoSuchMethodException, InvocationTargetException {
         File[] files = openPackage(getFileRepresentation(javaPackage));
         for (File file : files) {
+            // recursively open a sub-package
             if (file.list() != null) {
                 launch(getPackageFromFile(javaPackage, file), userData, signatures);
-            } else {
-                launchTestable(getClassFromPackage(javaPackage, file), userData, signatures);
+                continue;
+            }
+            // sanity filter out and skip non-class files (in case you have a dirty packaging)
+            if (!file.getName().contains(".class")) {
```

Comment:
This addition, is to avoid the unnecessary `ClassNotFoundException` thrown out if the packaging is not clean and having non-class files (as license files and module-info, etc).